src/org/usfirst/frc/team2335/robot/subsystems/Ultrasound.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
```

Comment:
I think I understand what Ben is saying though. We just need some explanation of what each thing does in this subsystem. Since it took us so long to finally design this, we want to be able to look back on this next year and understand what you did.